open-company/open-company-web
# Mobile Cmail attachments

Card: https://trello.com/c/p7t68DBN

Review after #918 

To test:
- from mobile enter Cmail
- add an attachment
- add another
- add some info to the post
- can you remove attachments?
- add another one
- post it
- now check the post in FoC, looking good?
- expand the post, still looking good? filename truncated well?
- now switch to desktop
- check FoC, looking good?
- expand the post, still looking good? filename truncated well if they need to?
- create another post with attachments
- did it work?
